sdk/include/ddk/ndis.h

```diff
@@ -4296,7 +4296,7 @@ NdisDprReleaseSpinLock(
 
 /*
  * VOID
- * NdisReadRegisterUshort(
+ * NdisWtiteRegisterUshort(
```

Comment:
Replace one typo with another? :p

Reply:
Well, it's closer now :P 
I'll fix that in the next commit